roles/k3s/master/tasks/main.yml

```diff
@@ -153,13 +152,20 @@
     owner: "{{ ansible_user }}"
     mode: "u=rw,g=,o="
 
-- name: Configure kubectl cluster to https://{{ apiserver_endpoint | ansible.utils.ipwrap }}:6443
+- name: Configure kubectl cluster to https://{{ apiserver_endpoint }}:6443
   command: >-
     k3s kubectl config set-cluster default
-      --server=https://{{ apiserver_endpoint | ansible.utils.ipwrap }}:6443
+      --server=https://{{ apiserver_endpoint }}:6443
```

Comment:
Why did you remove this?

Reply:
ok -- this is my mistake, 100%.  I thought I had the latest when I changed the file, but I didn't.  I need to redo this patch from the current.  This is totally me.  I worked on it, did what i needed, then copied the file in, instead of re-checking the master.  SMH.